larray/excel.py

```diff
@@ -453,9 +453,58 @@ def load(self, header=True, convert_float=True, nb_index=None, index_col=None):
                 return LArray(list_data)
 
     # XXX: remove this function?
-    def open_excel(filepath=None, **kwargs):
-        return Workbook(filepath, **kwargs)
+    def open_excel(filepath=None, new_file=False, **kwargs):
+        """
+        Open an Excel workbook.  
+        
+        Parameters
+        ----------
+        filepath: str
+            Path to an Excel file. The file must exist if `new_file` is not set to True.
+        new_file: bool, optional
+            If True and filepath represents a not existing Excel file, creates a new file.
+        \**kwargs
```

Comment:
other kwargs from Workbook should be copied here explicitly (OR open_excel should be deprecated telling users to use Workbook directly)